x-pack/plugins/cloud/index.d.ts

```diff
@@ -5,5 +5,7 @@
  */
 
 export interface CloudPlugin {
```

Comment:
Can we also update `x-pack/typings/hapi.d.ts` to make plugins optional?

Reply:
Done